test/example/main.tex

```diff
@@ -32,6 +32,19 @@
 \begin{align*}
 \end{align*}
 
+\def\nx{{\textnormal{x}}}
```

Comment:
This change was made to the test example file to provide a test case for a later improvement to the macro expansion code. I did not have the time to make the change as part of the PR, but I did have enough time to make this test case.